website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+
+***That’s good and all, but what does this look like for practitioners to use?*** 
+
+We’re building it with the following components[^1]:
+
+**Available Today**
+
+- `**[metric` node in `dbt-core` :](https://docs.getdbt.com/docs/building-a-dbt-project/metrics)** Similar to `models` or `sources` , this is a specific node type in `dbt-core`. It is the definition of a time-series aggregation over a table that supports 0+ dimensions. The resulting metadata is stored in the `manifest` just like `models` and referenced in the DAG.
+- `**[dbt_metrics` package:](https://github.com/dbt-labs/dbt_metrics)** this package provides macros that take in metric information regarding the `metrics` defined by users and then produces the necessary SQL to return the dataset containing that metric along with any other parameters defined (secondary calculations, specific dimensions, time grain, etc).
+- **[dbt Cloud Metadata API](https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview):** a GraphQL API which supports arbitrary queries over the metadata produced by dbt Cloud jobs. Contains metadata related to the accuracy, recency, configuration, and structure of the views and tables in the warehouse, as well as much more.
+
+**Forthcoming**
+
+- **dbt Server:** this component wraps dbt Core in a persistent server that is responsible for handling RESTful API requests for dbt operations. It’s a thin interface that is primarily responsible for performance and reliability in production environments.
+- **dbt Cloud proxy server:** this component enables dbt Cloud to intercept requests to a database and compile dbt-SQL into raw SQL that the database understands. It then returns the dataset produced by the raw SQL to the platform that sent it.
+
+![Untitled](/img/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer/semantic-layer-description.png)
+
+### Understanding how and when to use metrics?
+
+Let’s walk through a jaffle shop example of how you can use the components listed above to get started with the semantic layer today, as well as what it might look like once the forthcoming components are released.
+
+Say your CFO comes to you on a Monday morning and commands the data team to overhaul how the company is reporting on Revenue. Right now its a mess of tools and inconsistencies - Jim’s numbers are defined in Tableau and say one thing, Pam’s within Hex and say another! The CFO is frustrated with it and wants a cohesive experience across the company where everyone has the same numbers for revenue. 
```

Comment:
Does it feel like we're mixing metaphors here having Jim + Pam working on the jaffle shop (although I would certainly watch that reboot"). Maybe we replace with "the analytics engineer" / "their stakeholder" etc

Reply:
I will draw my line in the sand here to keep Jim and Pam in the jaffle shop! Let me rewrite it a bit to make it more clear that we're really pitching a reboot

Reply:
Updated